### DISCUSSION TITLE
Pls indicate default methods in "front facing doc"

### DISCUSSION THREAD

**Discussion by nrobidoux** [T8RIN/ImageToolbox#1373] [General]
E.g., when resizing, do you use the equivalent of -resize (separable) or -distort resize (EWA)? With what filters? Same as the ImageMagick defaults or ImageToolbox has its own? Is sigmoid space turned on by default? Resampling through RGB by default or some other space? Lots of options are listed but knowing ahead of time what the default is would be informative (and when it's something I invented I'd put in my CV).

**Comment by T8RIN** [comment]
What is chosen when the image is imported , this is the default
